### PR TITLE
add Git Tower cheat sheet.

### PR DESCRIPTION
**Pull request by luiscobits**
## What does this PR do?
Add resource
Git cheat sheet from Tower https://www.git-tower.com/blog/git-cheat-sheet/

This is the link to download the file: https://downloads.git-tower.com/content/cheat-sheet-bundle.zip

### Description

Git Cheat Sheet

### Why is this valuable (or not)?

It is a popular Git cheat sheet with the most important commands and workflows.

### How do we know it's really free?

We can download it directly from https://downloads.git-tower.com/content/cheat-sheet-bundle.zip

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -82,6 +82,7 @@
 * [Git Cheat Sheet](http://git.jk.gs) - Jan Krüger (PDF, SVG)
   * [Git Cheat Sheet](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet.pdf) (PDF)
   * [Git Cheat Sheet - extended](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet-v2.zip) (PDF)
+* [Git Cheat Sheet](https://downloads.git-tower.com/content/cheat-sheet-bundle.zip) - Tower (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:85]: is there a link that isn't zipped? Tower is an author?

Comment (davorpa) [reply more/free-programming-cheatsheets.md:85]: Frontend: https://www.git-tower.com/blog/git-cheat-sheet/

Comment (luiscobits) [reply more/free-programming-cheatsheets.md:85]: > is there a link that isn't zipped? Tower is an author?

No, they only have a zip link. It is only possible to add direct .pdf links?

Comment (davorpa) [reply more/free-programming-cheatsheets.md:85]: > > is there a link that isn't zipped? Tower is an author?
> 
> No, they only have a zip link. It is only possible to add direct .pdf links?

Not is limited, I thought

There are very PDF cheatsheets about this company inside. This includes Git, VSCode, Shell...

Comment (eshellman) [reply more/free-programming-cheatsheets.md:85]: It's not like we need another git cheatsheet. The thing about zips is it's easy to hide something bad inside.  

Weill close, feel free to reopen if the content is compelling.

**Pull request closed without merging**